adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,76 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_4_1']
+=== Release 1.4.1, July 2019
+
+[id='sec.1_4_1.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.17.1.
+* Added option to deploy UAA from the SCF Helm chart. In new CAP deployments that do not require a separate UAA deployment, setting `enable.uaa=true` in the Helm values automatically creates a UAA StatefulSet within the SCF deployment. This removes the manual step of copying the UAA_CA_CERT during setup. In this configuration UAA uses the same database as SCF, reducing the memory footprint of the cluster. This option should be left as `false` (default) for upgrades of existing CAP clusters.
+
+[id='sec.1_4_1.feature']
+==== Features and Fixes
+
+* Set the default value of AZ_LABEL_NAME to failure-domain.beta.kubernetes.io/zone
+* Simplified service accounts and pod security policies
+* Switched to log-cache for container metrics
+* Implemented a patch to squash Cloud Controller database migrations
+* Fixed version and SHA1 of cf-mysql-release 36.15.0
+* Fixed TLS issues in log-cache
+
+* Includes these {cf} component versions:
+
+** app-autoscaler: 1.2.1
+** bits-service: 
+** bpm: 
+** capi: 
+** cats: 7.11
+** cf-deployment: 7.11
+** cf-mysql: 36.15.0
+** cf-routing: 0.184.0
+** cf-sle12: 1.75.11
+** cf-smoke-tests: 40.0.44
+** cf-syslog-drain: 8.1
+** cf-usb: 1.0.1
+** cflinuxfs2: 1.281.0
+** cflinuxfs3: 0.108.0
+** credhub: 2.1.2
+** diego: 2.25.0
+** eirini: 0.0.4
+** garden-runc: 1.17.2
+** groot-btrfs: 1.0.4
+** kubectl: 1.9.6
+** loggregator: 104.4
+** loggregator-agent: 3.2
+** nats: 26
+** nfs-volume: 1.7.6
+** postgres-release: 26
+** scf-helper: 1.0.2
+** cf-acceptance-tests: 
```

Comment:
Should we remove this? If this is same as CATS then this would be 7.11

Reply:
removed